### PR TITLE
Add in publish command and check command that informs user if images are missing

### PR DESCRIPTION
**Pull request by pbeshai**
If you want to include these, here's the PR that addresses #22. Adds a dimmed warning message if missing thumbnail.png, preview.png|jpg.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -79,7 +79,40 @@ gulp.task('stylus', function() {
 
 })
 
+/**
+ * Helper function that checks if a file exists in the current working directory.
+ *
+ * @param {String} filename The file to check
+ * @return {Boolean} true if the file exists, false otherwise
+ */
+const fileExists = (filename) => {
```
Comment (gabrielflorit) [inline src/index.js]: This is solid documentation - wish all PR content came with this.

**Comment by pbeshai** [comment]
I've updated this to add in two commands:
- `blockup check` -- checks for missing files (thumbnail.png, preview.png|jpg, README.md)
- `blockup publish` -- publishes directly to git via gistup. Gives interactive prompt for setting title.

#### Publish!

image

#### No to missing files

image

#### Yes to missing files being ok, but it's a git repo

image

#### Interaction demo

interactiondemo

**Comment by gabrielflorit** [comment]
This looks fabulous!!! I might have to get back to you in a bit, elections work is overwhelming right now. Sorry!

**Comment by gabrielflorit** [comment]
Wow man this is outstanding.

**Pull request merged**